ClothesApp/Infrastructure/Repositories/OrderItemsRepository.cs

```diff
@@ -0,0 +1,64 @@
+using Application.Interfaces.Repositories;
+using Domain.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace Infrastructure.Repositories;
+
+public class OrderItemsRepository : IOrderItemsRepository
+{
+    private readonly ClothesAppContext _dbContext;
+
+    public OrderItemsRepository(ClothesAppContext dbContext)
+    {
+        _dbContext = dbContext;
+    }
+
+    public async Task<OrderItem> Insert(OrderItem orderItem)
+    {
+        _dbContext.OrdersItems.Add(orderItem);
+        await _dbContext.SaveChangesAsync();
+
+        return orderItem;
+    }
+
+    public async Task<OrderItem> Update(OrderItem orderItem)
+    {
+        _dbContext.OrdersItems.Update(orderItem);
+        await _dbContext.SaveChangesAsync();
+
+        return orderItem;
+    }
+
+    public Task<IList<OrderItem>> GetAll()
+    {
+        throw new NotImplementedException();
```

Comment:
same thing, too much not implemented methods, either implement them or don't implement generic repo